packages/js/client/src/PolywrapClient.ts

```diff
@@ -633,52 +615,59 @@ export class PolywrapClient implements Client {
   @Tracer.traceMethod("PolywrapClient: _loadWrapper", TracingLevel.High)
   private async _loadWrapper(
     uri: Uri,
+    resolutionContext?: IUriResolutionContext,
     options?: Contextualized
   ): Promise<Wrapper> {
     Tracer.setAttribute("label", `Wrapper loaded: ${uri}`, TracingLevel.High);
 
-    const { wrapper, uriHistory, error } = await this.resolveUri(uri, {
+    if (!resolutionContext) {
+      resolutionContext = new UriResolutionContext();
+    }
```

Comment:
Yes, since the context is an optional argument (and makes sense to keep it that way) and we need it to get history to show in the error message in case of an error